pkg/awsutils/imds.go

```diff
@@ -177,12 +177,31 @@ func (imds TypedIMDS) getCIDRs(ctx context.Context, key string) ([]net.IPNet, er
 	return cidrs, nil
 }
 
+func (imds TypedIMDS) getPrefixes(ctx context.Context, key string) ([]string, error) {
```

Comment:
Yes, it is identical. Reason I added was for readability but thinking about it again, it should be fine to replace with GetLists since the caller is GetLocalIPv4Prefixes.